### ISSUE TITLE
`Err` value: InvalidTickLen("xai\u{307}")'

### ISSUE THREAD

**Issue by zhiying8710** [okx/ord#87]
```
2023-08-06T08:14:18.951224922+00:00 INFO ord::index::updater - Block 800453 at 2023-07-27 06:52:27 UTC with 3366 transactions…
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: InvalidTickLen("xai\u{307}")', src/brc20/types/tick.rs:40:59
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
2023-08-06T08:15:14.653583764+00:00 INFO ord::index::updater - Block receiver disconnected: sending on a closed channel
```
hang on this error, how to fix it?

**Comment by wanyvic** [comment]
> ```
> 2023-08-06T08:14:18.951224922+00:00 INFO ord::index::updater - Block 800453 at 2023-07-27 06:52:27 UTC with 3366 transactions…
> thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: InvalidTickLen("xai\u{307}")', src/brc20/types/tick.rs:40:59
> note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
> 2023-08-06T08:15:14.653583764+00:00 INFO ord::index::updater - Block receiver disconnected: sending on a closed channel
> ```
> 
> hang on this error, how to fix it?

We are aware of the issue and are currently working on fixing it. After conducting regression testing, we will merge #88.

**Comment by wanyvic** [comment]
The issue has been fixed by #88.
